org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/xpect/ConstructorDelegation.wlk.xt

```diff
@@ -19,7 +19,7 @@ class B {
 }
 
 class C inherits B {
-	// XPECT errors --> "Invalid constructor call. Does Not exist" at "super()"
+	// XPECT errors --> "Invalid constructor call. It does not exist. Valid constructors are: super(_x)" at "super()"
```

Comment:
Ok, but I'll check whether it is a WSelfDelegatingConstructor or WSuperDelegatingConstructor.